src/v3/global.css

```diff
@@ -598,7 +598,7 @@
   font-style: normal;
   font-stretch: normal;
   letter-spacing: -0.12px;
-  line-height: 16px;
+  line-height: 14px;
```

Comment:
@eloyEntony: have already have confirmation about this change from the designers?

Reply:
@rhahao yes, it has been discussed. Hopefully this change won't cause more further issues on next stages 😀